extensions/wikia/PremiumPageHeader/PremiumPageHeaderController.class.php

```diff
@@ -41,22 +320,38 @@ private function getExplore() {
 		}, $explore );
 
 		return [
-			'text' => wfMessage( 'pph-explore' )->escaped(),
+			'text' => wfMessage( 'pph-explore' )->inContentLanguage()->escaped(),
 			'children' => array_filter( $children, function ( $child ) {
 				return !empty( $child );
 			} )
 		];
 	}
 
-	private function getDiscuss() {
+	private function getDiscuss(): array {
 		global $wgEnableDiscussionsNavigation, $wgEnableDiscussions, $wgEnableForumExt;
 
-		$href = !empty( $wgEnableDiscussionsNavigation ) && !empty( $wgEnableDiscussions ) && empty( $wgEnableForumExt ) ?
-			'/d' : Title::newFromText( 'Forum', NS_SPECIAL )->getLocalURL();
+		$href =
+			!empty( $wgEnableDiscussionsNavigation ) && !empty( $wgEnableDiscussions ) &&
```

Comment:
I'd assume that `explore` navigation tab will be provided externally, so we won't be checking this anymore.